docs/installation/windows.md

```diff
@@ -93,13 +94,16 @@ curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
 echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
 ```
 
-Then you can simply issue:
+Since we do not have `Node.js` installed yet, we will be installing Yarn without
+the default `Node.js` with the following command:
 
 ```shell
-sudo apt-get update && sudo apt-get install yarn
+sudo apt update && sudo apt install --no-install-recommends yarn
 ```
 
-Make sure that Yarn is installed with `yarn -v`
+Make sure that Yarn is installed with the command `yarn -v`. It should print
```

Comment:
I imagine this is just a warning? At this point, yarn is installed, just that it won't work until we nvm install, correct?

Reply:
Yes, it is! It will just print the warning. I will change the sentence to become explicit about that.